react/components/organisms/listing/listing.js

```diff
@@ -60,10 +60,13 @@ export class Listing extends Component {
             itemsOffset: 0,
             filters: this.props.filtersValue,
             loading: false,
+            searchLoaded: false,

```

Comment:
Why is this needed?

Reply:
because it will trigger multiple times if this is not done this way. We already have this pattern in other places in Robin